plugins/CoreHome/templates/_applePinnedTabIcon.twig

```diff
@@ -1,4 +1,4 @@
 {% if isCustomLogo and customFavicon is defined and customFavicon %}
 {% else %}
-	<link rel="mask-icon" href="plugins/CoreHome/images/applePinnedTab.svg" color="#C81319">
+	<link rel="mask-icon" href="plugins/CoreHome/images/applePinnedTab.svg" color="#d4291f">
```

Comment:
The svg image must be completely black. The color is written into in the HTML-Head. Here some more information about the Tab Icon: [Apple Developer Library](https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html)